cloudanalysis/get_fv3sar_bk_parallel_mod.f90

```diff
@@ -613,6 +633,14 @@ subroutine release_mem_fv3sar
      if(allocated(ges_qnr)) deallocate(ges_qnr)
      if(allocated(ges_qni)) deallocate(ges_qni)
      if(allocated(ges_qnc)) deallocate(ges_qnc)
+     if(l_cld_uncertainty) then
+       write(6,*) 'core', 1 ,', release memory for hydrometeor uncertainties'
```

Comment:
This only applies to core 1?

Reply:
Other cores will have the same information. It is better to just say:
write(6,*)  ' release memory for hydrometeor uncertainties'

Reply:
It turns out that I don't need the uncertainty fields in the get_fv3sar_bk_parallel_mod.f90 code, since I overwrite the hydrometeor fields in the cloud_analysis before calling the update code.  I will remove all the unc_* fields and the allocate/deallocate steps.

Reply:
True, this will write out to each core, so it shouldn't say core '1' each time.  I can make it more general.

Note that this also happens in the current code when writing out "release memory" messages, e.g., in
NCO_dirs/stmp/2023072102/nonvar_cldanl/stdout_cloudanalysis.d0*, each file ends with:

 gsdcloudanalysis: success in modifying hydrometeors for stratiform clouds
 core           1 ,finished, now release memory or hydrometeors
 core           1 ,finished, now release memory for fix variables
 CLDcount  0.450000000000000           837617        4887       63716
      157700
 ========================================
 gsdcloudanalysis: generalized cloud analysis finished:           2
 ========================================

I could update those messages as well?